code/modules/surgery/organs/organ_external.dm

```diff
@@ -699,6 +709,9 @@ Note that amputating the affected organ does in fact remove the infection from t
 					throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),30)
 				dir = 2
 			return
+		else
```

Comment:
This is in the droplimb proc, which is only called with an argument of `disintegrate` other than DROPLIMB_EDGE, on the `fail_on_full_damage = 2` limbs, which WERE only the slime people limbs, before they were made normal again because people griped about them without playing with them.

I believe this was to fix the limb simply falling to the floor if they got vaporized or smashed, even though those situations don't ever come up in this current codebase.